app/cmd/config_remove.go

```diff
@@ -30,7 +30,8 @@ var configRemoveCmd = &cobra.Command{
 func removeJenkins(name string) (err error) {
 	current := getCurrentJenkins()
 	if name == current.Name {
-		err = fmt.Errorf("You cannot remove current Jenkins")
+		err = fmt.Errorf("You cannot remove current Jenkins config, if you want remove current Jenkins config, you can select the other Jenkins config before remove")
```

Comment:
> You cannot remove current Jenkins config, if you want remove current Jenkins config, you can select the other Jenkins config before remove

Please consider the blew revise:

`You cannot remove the current Jenkins config item. If you want to remove it, please select another one before remove.`

Plus, please try not to make a very long line.